mongo/options/createcollectionoptions.go

```diff
@@ -65,6 +79,20 @@ func (tso *TimeSeriesOptions) SetGranularity(granularity string) *TimeSeriesOpti
 	return tso
 }
 
+// SetBucketMaxSpan sets the value for BucketMaxSpan.
+func (tso *TimeSeriesOptions) SetBucketMaxSpan(dur time.Duration) *TimeSeriesOptions {
```

Comment:
@kevinAlbs @qingyang-hu I changed the setters to match the fields, from "SetBucketMaxSpanSeconds" to "SetBucketMaxSpan".